jitsi_meet/android/src/main/kotlin/com/gunschu/jitsi_meet/RNJitsiMeetConferenceOptions.java

```diff
@@ -0,0 +1,338 @@
+package com.gunschu.jitsi_meet;
+
+import android.os.Bundle;
+import android.os.Parcel;
+import android.os.Parcelable;
+
+import java.net.URL;
+
+
+/**
+ * This class represents the options when joining a Jitsi Meet conference. The user can create an
+ * instance by using {@link RNJitsiMeetConferenceOptions.Builder} and setting the desired options
+ * there.
+ *
+ * The resulting {@link RNJitsiMeetConferenceOptions} object is immutable and represents how the
+ * conference will be joined.
+ */
+public class RNJitsiMeetConferenceOptions implements Parcelable {
```

Comment:
Could we somehow add the RN classes from the react native plugin repo, so that we don't need to copy everything? Maybe we could fork the repo (either straight to this repo, if possible, or I can also fork it with BindrUK on GitHub) and change the small things that we need to change.